install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        repo="https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"
+        ;;
+    *)
+        echo "Not supported OS release."
+        return -1;;
+    esac
+    path=${repo%/*}/
+    if zypper lr --details | cut -d'|' -f9 | grep $path; then
+        echo "$path repository alreadt exists. Skipping."
```

Comment:
Small typo here, I think you meant 'already'

Reply:
Good catch, thanks!